tidb-cloud/import-with-mysql-cli.md

```diff
@@ -59,12 +63,13 @@ Do the following to import data from an SQL file:
 2. Use the following command to import data from the SQL file:
 
     ```bash
-    mysql --comments --connect-timeout 150 -u '<your_username>' -h <your_cluster_host> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=<your_ca_path> -p <your_password> < product_data.sql
+    mysql --comments --connect-timeout 150 -u '<your_username>' -h <your_cluster_host> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=<your_ca_path> -p<your_password> < product_data.sql
     ```
 
 > **Note:**
 >
-> The default database name used here is `test`, and you can either manually create your own database or use the `CREATE DATABASE` command in an SQL file.
+> - The default database name used here is `test`, and you can either manually create your own database or use the `CREATE DATABASE` command in an SQL file.
+> - The MySQL v9.0 client on macOS cannot correctly load the `mysql_native_password` plugin, causing the error `ERROR 2059 (HY000): Authentication plugin 'mysql_native_password' cannot be loaded` when connecting to TiDB. To address this issue, it is recommended to install and use the MySQL v8.0 client to connect to TiDB.
```

Comment:
```suggestion
```

Reply:
The issue occurred only for customers who installed the default MySQL Client via Homebrew (due to a missing plugin in the package, specific to the macOS distribution via Homebrew). As of today, users installing MySQL Client via Homebrew will receive version 9.1 by default and should no longer encounter this issue.